bacon/controller/CoreLogic.swift

```diff
@@ -115,4 +115,18 @@ class CoreLogic: CoreLogicInterface {
     func addChildTag(_ child: String, to parent: String) throws -> Tag {
         return try tagManager.addChildTag(child, to: parent)
     }
+
+    func removeChildTag(_ child: String, from parent: String) throws {
```

Comment:
Should parent be `String?` For TagManager all the parent types are String optionals

Reply:
But I guess since the method name is removeChildTag it assumes that it will have a valid parent. In that case, maybe can put an assert() statement?